armi/operators/operator.py

```diff
@@ -30,6 +30,14 @@
 import re
 import shutil
 import time
+import collections
+from tabulate import tabulate
```

Comment:
style: there should be a blank line between 33 and 34 (pylint should tell you this)